services/core/shared/core/compat/sdk_v5/blocks.js

```diff
@@ -13,13 +13,17 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const { computeMinFee } = require('@liskhq/lisk-transactions-v5');
+const { CacheRedis, Logger } = require('lisk-service-framework');
 
 const coreApi = require('./coreApi');
-const {
-	getBlockchainTime,
-	validateTimestamp,
-} = require('../common');
+const config = require('../../../../config');
+
+// const { indexTransactions } = require('./transactions');
+const { getApiClient, parseToJSONCompatObj } = require('../common');
+const { knex } = require('../../../database');
+
+const logger = Logger();
+const blocksCache = CacheRedis('blocks_SDKv5', config.endpoints.redis);
```

Comment:
```suggestion
const blocksCache = CacheRedis('blocks', config.endpoints.redis);
```
I think, there is no need to define as a `blocks_SDKv5` because given logic is already implemented in `compat` layer (SDKv5). This would be more generic `const blocksCache = CacheRedis('blocks', config.endpoints.redis);`